mptt/managers.py

```diff
@@ -94,15 +95,13 @@ def get_query_set(self, *args, **kwargs):
 
         This method can be removed when support for Django < 1.6 is dropped.
         """
-        return super(TreeManager, self).get_query_set(*args, **kwargs).order_by(
-            self.tree_id_attr, self.left_attr)
+        return TreeQuerySet(self.model).order_by(self.tree_id_attr, self.left_attr)
```

Comment:
I think this should use `TreeQuerySet(self.model, using=self._db)`... for multi-db support.